public/stations-along-route/index.html

```diff
@@ -57,7 +57,7 @@ <h2>Alternative station data</h2>
         <li class="info-show">
           Show alternative stations
           <label class="switch" id="alternative">
-            <input type="checkbox" />
+            <input type="checkbox" onclick="return false;" id="check" />
```

Comment:
Not sure what is happening with listeners for this component :) What do you try to achieve?